src/containers/helpers/permissionsHelper.js

```diff
@@ -1,7 +1,7 @@
-export const displayPermissions = verbs => {
+export const displayPermissions = (verbs, t) => {
```

Comment:
I think the implementation could be simplify.

=> no needed to give "t" here.

the component using this helper can do the t(displayPermissions)

Reply:
@trollepierre I feel that this kind of API a bit dangerous ; if I want to use this helper elsewhere and I do not pay attention, I will just call `displayPermissiosn(verbs)` and cause a crash as `t` will be undefined. Also, this will not work if I want to use it outside of a component. For me a helper should rely only on its own arguments

Reply:
Vu avec Paul :
=>

la fonction doit juste retourner une clé de traduction (comme par ex : 'Permissions.readAndWrite') et le composant pourra faire appel à la fonction `t` du résultat.